Kong/docs.konghq.com
# Remove ArecaBay plugin

### Description

This is a followup to https://github.com/Kong/docs.konghq.com/pull/5339.
The Imperva API Security plugin replaces this old ArecaBay plugin.

### Testing instructions

Check that links to `/hub/arecabay/microsensor` redirect to `/hub/imperva/imp-appsec-connector/`.

https://deploy-preview-5363--kongdocs.netlify.app/hub/arecabay/microsensor

### Checklist
